sqlglot/generator.py

```diff
@@ -475,7 +488,7 @@ def where_sql(self, expression):
 
     def window_sql(self, expression):
         this_sql = self.sql(expression, "this")
-        partition = expression.args.get("partition")
+        partition = expression.args.get("partition_by")
```

Comment:
let's not change this for now

Reply:
discussed offline, will keep the distinction here.